alpheios_nemo_ui/data/assets/js/text.js

```diff
@@ -1,10 +1,12 @@
 $(document).ready(function(){
-  $("[data-toggle=popover]").each(function(i, obj) {
-    $(this).popover({
-      html: true,
-      content: function() {
-        return $(this).next('.popover-content').html();
-      }
-    });
-  });
+  $("[data-toggle=popover]").each(showNotes)
 });
+
```

Comment:
@balmas , I tried to find the difference between the code  (except that it was moved to the external function) and failed! 
could you help me and tell - what exactly it should fix?
when we scroll and add new passages - it doesn't add popover handler to new added ones?